modules/core/src/v2/environments.ts

```diff
@@ -94,7 +94,9 @@ const mainnetBase: EnvironmentTemplate = {
   signingAddress: '1BitGo3gxRZ6mQSEH52dvCKSUgVCAH4Rja',
   serverXpub: hardcodedPublicKeys.serverXpub.prod,
   hsmXpub: hardcodedPublicKeys.hsmXpub.prod,
-  blockchairBaseUrl: 'https://api.blockchair.com/bitcoin',
+  blockchairBaseUrl: (coin: String) => {
```

Comment:
use `blockchairBaseUrl(coin: string) { ... }`

better `getBlockchairBaseUrl(coin)` to highlight that it is a function

Reply:
if it is a function it should better move out of `config` or `environments`

this is better suited for  `blockchairApi.ts` with the a signature `getBaseUrl(coin: string, testnet: boolean)` 

the `coin: String` is a bit misleading - typically that indicates our three-character shorthands `btc` `eth` etc, while here we only should allow strings like `bitcoin` `ethereum` `litecoin` etc

We should validate the argument to make sure it is one of those